src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -99,10 +99,15 @@ public class PullRequestHandler : IHttpHandler
             {
                 case "master":
                 case "origin/master":
+                case "origin/infrastructure":
                     success = SyncToBranch(branchString, result);
                     break;
+                case "origin/release/1.0.0-rc2":
+                    context.Response.Write(string.Format("Branch name specified, '{0}' will not be synced by this service<br/>", HttpUtility.HtmlEncode(branchString)));
+                    success = true; 
```

Comment:
In case I did not make it clear - we don't want to enable any new testing for rc2. Please undo the changes about rc2.